src/data/finset/basic.lean

```diff
@@ -2241,6 +2242,7 @@ begin
   { simp [finset.singleton_inter_of_mem h] },
 end
 
+@[simp]
 theorem card_erase_of_mem [decidable_eq α] {a : α} {s : finset α} :
   a ∈ s → card (erase s a) = pred (card s) := card_erase_of_mem
```

Comment:
`multiset.card_erase_of_mem` is already a simp lemma, so +1 from me

Reply:
Same here. 👍